recipes/dcmtk/all/conanfile.py

```diff
@@ -91,101 +82,115 @@ def requirements(self):
         if self.options.with_zlib:
             self.requires("zlib/1.2.13")
         if self.options.with_openssl:
-            if self.settings.os == "Windows":
-                # FIXME: CMake configuration fails to detect Openssl 1.1 on Windows.
-                self.requires("openssl/1.0.2u")
-            else:
-                self.requires("openssl/1.1.1t")
+            self.requires("openssl/1.1.1t")
```

Comment:
We are already allowing version ranges for openssl, if this recipe will build with openssl 3, it is possible:
```suggestion
            self.requires("openssl/[>=1.1 <4]")
```